mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/model/Attachment.java

```diff
@@ -29,7 +33,7 @@ public Attachment(InputStream stream) {
         this.stream = stream;
     }
 
-    public InputStream getStream() {
-        return stream;
+    public InputStream getStream() throws IOException {
+        return new ByteArrayInputStream(IOUtils.toByteArray(stream));
```

Comment:
Ok, but then we may have to consume the inputStream, and then what is the added value of keeping an InputStream in the API?
We may use a byte array, WDYT?